src/porting_android.cpp

```diff
@@ -157,7 +157,7 @@ void showTextInputDialog(const std::string &hint, const std::string &current, in
 			jhint, jcurrent, jeditType);
 }
 
-void showComboBoxDialog(const std::string optionList[], s32 listSize, s32 selectedIdx)
+void showComboBoxDialog(const std::string *optionList, s32 listSize, s32 selectedIdx)
```

Comment:
warning we can now pass null there

Reply:
As far as I know `T[]` and `T*` are actually the exact same in function arguments, and both nullable.

Reply:
sfan5 is right: https://stackoverflow.com/questions/10760893/c-vs-as-a-function-parameter